src/LaunchDarkly.EventSource/EventSourceService.cs

```diff
@@ -99,30 +102,26 @@ CancellationToken cancellationToken
 
                 using (var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false))
                 {
-                    bool needManualTimeout = false;
-                    try
-                    {
-                        // Some Stream subclasses on some platforms have their own read timeout implementation; others
-                        // don't. The only way to find out is to try to set it and see if you get an exception. If we
-                        // get an exception, then ReadTimeout won't work and we'll need to implement our own timeout.
-                        stream.ReadTimeout = (int)_configuration.ReadTimeout.TotalMilliseconds;
```

Comment:
This logic turned out to be useless because the Stream subclass that you get for a network socket, for whatever reason, is not one of the ones that supports a read timeout natively.